testing/binarySearch.js

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai'
+import binarySearch from '../functions/binarySearch'
+
+
+describe('binarySearch()', function(){
+  context('take a sorted array and a targeted value', function(){
```

Comment:
Can you also write a test case for when the function doesnt find the target value?